app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split(" ")) != 3 or command.split(" ")[0] != "mv":
+        print("wrong command")
+        return
+
+    file_name, new_path = command.split(" ")[1:]
+
+    with open(file_name, "r") as sorce:
```

Comment:
There's a typo in the variable name `sorce`. It should be corrected to `source` for clarity and accuracy.